scipy/interpolate/tests/test_rgi.py

```diff
@@ -453,6 +453,10 @@ def f(x, y):
         (5, lambda x, y, z, a, b: 2 * x ** 3 + 3 * y ** 2 - z + a * b),
     ])
     def test_descending_points_nd(self, method, ndims, func):
+
+        if ndims == 5 and method in {"cubic", "quintic"}:
+            pytest.skip("too slow; OOM (quintic); or nearly so (cubic)")
```

Comment:
Could we mark this with an `@slow`/`@xslow` decorator?

Reply:
First, I don't know how to mark one value of a parameter in a parametrized test individually. This is probably doable with some bending over backwards but I'd argue it's not needed really: 
- skipping does not leave a hole in coverage, it's covered for `method='slinear'`
- the test is simply not a good unit test: it hogs memory, throttles the CPU, and eventually OOMs for me locally.

So this test really illustrates the need for keeping the "legacy" methods which work just fine for this example.

Reply:
That's fair. I'll mark this as resolved.